.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+
+app-image: &app-image circleci/ruby:2.5.1-node
+app-env: &app-env
+  BUNDLE_PATH: vendor/bundle
```

Comment:
We use `app-image && app-env` only once. So maybe the definition is not needed and can be used directly. But on the otherside its consistent to the other projects. Got not strong opinion here :D

Reply:
My argument was also about consistency. And it's easier and safer to copy-paste in this case 🙂